playground/webpack.config.js

```diff
@@ -97,7 +97,23 @@ module.exports = (env = {production: false}) => ({
     rules: [
       {
         test: /\.md$/,
-        use: [{loader: `${__dirname}/webpack/parseMarkdown.js`}],
+        use: [
+          {
```

Comment:
How does this work? You parse MD files with babel first?

Reply:
Other way around.  [Loaders are ran bottom to top](https://webpack.js.org/concepts/loaders/#configuration).

Order of operations is:
- Use the loader defined in parseMarkdown to turn the md files into  a single es6 module
- run that module through babel using babel-loader.

Same sort of thing as below - we run ts files through ts-loader then babel-loader.